cmd/gateway/main.go

```diff
@@ -381,6 +402,46 @@ func startGateway() {
 		tlsCertificate = loadedTlsCertificate
 	}
 
+	if config.cbPassAwsId != "" {
+		if config.cbPassAwsRegion == "" {
```

Comment:
We should have some restrictions around rejecting if the user specifies cb-user or cb-pass when using secrets to populate those fields.